ScienceJournal/UI/ObserveViewController.swift

```diff
@@ -898,10 +902,16 @@ open class ObserveViewController: ScienceJournalCollectionViewController, ChartC
 
   // Animates the display of the time axis view.
   func showTimeAxisView(_ isVisible: Bool) {
-    UIView.animate(withDuration: 0.5, animations: {
-      self.timeAxisController.timeAxisView.alpha = isVisible ? 1 : 0
-    }) { (_) in
-      self.adjustContentInsets()
+    if FeatureFlags.isActionAreaEnabled {
+      self.transitionCoordinator?.animate(alongsideTransition: { _ in
+        self.timeAxisController.timeAxisView.alpha = isVisible ? 1 : 0
+        self.adjustContentInsets()
+      }, completion: nil)
+    } else {
+      UIView.animate(withDuration: 0.4) {
```

Comment:
Note that I changed the duration from `0.5` to match the rest of the app's animation timings.